kernel_gateway/tests/test_jupyter_websocket.py

```diff
@@ -252,7 +252,7 @@ def test_auth_token(self):
         except Exception as ex:
             self.assertEqual(ex.code, 401)
         else:
-            self.assert_(False, 'no exception raised')
+            raise AssertionError('no exception raised')
```

Comment:
`self.assert_` is deprecated, but replacing it with `self.assertTrue(False, ...)` looks ugly. So I've replaces it with just raising an exception.